includes/admin/payments/orders.php

```diff
@@ -33,6 +33,21 @@ function edd_order_details_publish( $order ) {
 		<div class="edit-post-header">
 
 			<div class="edit-post-header__settings">
+				<?php if ( edd_is_add_order_page() ) : ?>
+					<div class="edd-send-purchase-receipt">
+						<div class="edd-form-group">
+							<div class="edd-form-group__control">
+								<input type="checkbox" name="edd_order_send_receipt" id="edd-order-send-receipt" class="edd-form-group__input" value="1" />
+
+								<label for="edd-order-send-receipt">
+								<?php esc_html_e( 'Send Purchase Receipt', 'easy-digital-downloads' ); ?>
+								</label>
+								<span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>Send Receipt</strong>: checking this box will send the purchase receipt to the selected customer.', 'easy-digital-downloads' ); ?>"></span>
```

Comment:
While we're in here, could we go ahead and escape the help-tip text on output? Since it has a `strong` tag in it, it may need to be split into to separate translation strings, not sure about that, since `esc_html_e` would break that.

Reply:
I think we can just remove the `<strong>Send Receipt</strong>:` prefix in the tooltip. The tooltip is given the context of the label, so I don't think it's necessary to repeat it.

I think the tooltip should change to `Checking this box will email` (replacing `send` with `email`).

Reply:
For the sake of consistence, at least on this page, should I change the `edd-help-tip` for Unlimited Downloads, too?

Reply:
@LisaCee Sure! This instance still needs @robincornett's original suggestion of `esc_html_e()` too.